at_client/src/main/java/org/atsign/client/cli/Register.java

```diff
@@ -0,0 +1,45 @@
+package org.atsign.client.cli;
+
+import java.util.Scanner;
+import org.atsign.client.util.RegisterUtil;
+
+public class Register {
+    public static void main(String[] args) throws Exception {
+        if (args.length == 0) {
+            System.err.println("Usage: Register <email@email.com>");
+            System.exit(1);
+        }
+
+        String email = args[0];
+        String otp;
+        String validationResponse;
+        String cramSecret;
+
+        Scanner scanner = new Scanner(System.in);
+        Onboard onboardCli = new Onboard();
+        RegisterUtil registerUtil = new RegisterUtil();
+        String atsign = registerUtil.getFreeAtsign();
```

Comment:
Pls check how the rest of the code handles the exceptions. Let's say if there is no network or an exception is thrown by the REST api, what do we show on the CLI?

Reply:
Anything other than an HTTP_200 is thrown as an exception. I have added print statements for HTTP_200, in case the request is accepted but other issues arise such as wrong otp.